### ISSUE TITLE
Is this test failure expected?

### ISSUE THREAD

**Issue by prescod** [SFDO-Tooling/sfdoc#193]
Some URL seems to have changed and is causing a test failure.

```
AssertionError: 'http[31 chars].com/apex/Hub_KB_ProductDocArticle?id=123&prev[16 chars]=APP' != 'http[31 chars].com/knowledge/publishing/articlePreview.apexp?id=123'

- https://sb-testcommunity.cs70.force.com/apex/Hub_KB_ProductDocArticle?id=123&preview=true&channel=APP
+ https://sb-testcommunity.cs70.force.com/knowledge/publishing/articlePreview.apexp?id=123
```

**Comment by davisagli** [comment]
Looks like the test needs to be updated to match @dextermilo's update in #190

**Comment by dextermilo** [comment]
Indeed. In addition, it had occurred to me to move this to something configurable as it may want to change. #195 

Sorry for the noise.

**Comment by dextermilo** [comment]
closed by #195
